src/utils/endUserMessages.js

```diff
@@ -0,0 +1,24 @@
+export default {
+  connectionError: 'Could not connect to the server. Check your internet connection and refresh the page.',
```

Comment:
I think for constants we have lately been using full uppercase like CONNECTION_ERROR